src/components/Map/Map.tsx

```diff
@@ -64,20 +64,57 @@ function ConfigureMap({ center, zoom }: MapConfigProps) {
   return null;
 }
 
+// function Form() {
+//   const [zipcode, setZipcode] = useState('')
+//   const [radius, setRadius] = useState('')
+//   return <>
+//     <form>
+//       <input
+//         type='text'
+//         name='zip'
+//         placeholder='Enter zipcode'
+//         value={zipcode}
+//         >
+
+//       </input>
+//       <input
+//         type='text'
+//         name='radius'
+//         placeholder='Enter radius'
+//         value={radius}>
+//       </input>
+//       <button>Search</button>
+//     </form>
+    
+//   </>
+  
+// }
+
 function Map() {
-  //   console.log('farmersMarkets', farmersMarkets);
 
+  //   console.log('farmersMarkets', farmersMarkets);
+  
   const [selectedMarketByZip, setSelectedMarketByZip] =
     useState<selectedMarketProps | null>(null);
+  const [zipcode, setZipcode] = useState('')
+  const [radius, setRadius] = useState('')
+  const [map, setMap] = useState<L.Map | null>(null)
+  const [searchClicked, setSearchClicked] = useState(false)
 
   useEffect(() => {
-    getMarkets()
+    getMarkets(zipcode, radius)
       .then(data => {
         setSelectedMarketByZip(data);
         console.log('data from API', data);
+
+        if (data && data.length > 0 && map) {
+          const firstMarket = data[0]
```

Comment:
Just for clarification for Ann. This is taking the first market in the array and using its longitude and latitude as the flyTo point.